.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+os:
+  - linux
+addons:
+  apt:
+    packages:
+      - autoconf
+      - automake
+      - build-essential
+      - clang
+      - curl
+      - libboost-dev
+      - libboost-program-options-dev
+      - libogg-dev
+      - ninja-build
+before_install:
+  - curl -L -o patchelf-0.8.tar.gz https://github.com/NixOS/patchelf/archive/0.8.tar.gz
+  - tar -xzf patchelf-0.8.tar.gz
+  - pushd patchelf-0.8 ; ./bootstrap.sh ; ./configure ; make ; popd
```

Comment:
Nit: You probably want `&&` instead of `;` in case something goes awry. It should be safe to run `make install` as well, since these are just containers anyway. Then you can drop the environment variable below.